content/en/getting_started/synthetics/private_location.md

```diff
@@ -0,0 +1,60 @@
+## Set up your first private location
+
+Private locations allow you to monitor internal-facing applications or any private URLs that aren’t accessible from the public internet. They can also be used to create a new custom Synthetics location.
+
+The private location worker is shipped as a Docker container, so it can run on a Linux based OS or Windows OS if the Docker engine is available on your host and can run in Linux containers mode.
+
+By default, every second, your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test depending on the frequency defined in the configuration of the test, and returns the test results to Datadog’s servers.
+
+Once you created a private location, configuring a [Synthetics API test][1] from a private location is completely identical to the one of Datadog managed locations.
+
```

Comment:
And here start a second section called "configure your PL" or "set up your pl" or "get your pl started" or sthg